### DISCUSSION TITLE
Default behavior of removeSelectables()

### DISCUSSION THREAD

**Discussion by arleybls** [ThibaultJanBeyer/DragSelect#114] [General]
I am playing around with the library and noticed that revomeSelectables() only lasts for the next selection action, after that, the selectable class is added right back to the element. Is that the expected behaviour, or wasn't removeSelectables() supposed to be a permanent remove instead?

**Comment by ThibaultJanBeyer** [comment]
Hi @arleybls that sounds like a bug. Can you please share an example so I can try to reproduce it? Thank you!

**Comment by arleybls** [comment]
hi @ThibaultJanBeyer 
You can give it a try using the console in your codepen, just give a var to the object and, for example:
`myobj.removeSelectables(document.getElementsByClassName('four'));`

At this point you will see the **ds-selectable** class being removed from the element, but right after you click on a blank area the class is added back thus making the element 'four' selectable again.

**Comment by arleybls** [comment]
Hi @ThibaultJanBeyer have had the chance to look into this?

**Reply by ThibaultJanBeyer** [reply]
Hi @arleybls sorry I must have overlooked this. Is this still the case? Please [file a bug report](https://github.com/ThibaultJanBeyer/DragSelect/issues/new?assignees=&labels=&template=bug_report.md) and make sure to add a code example so we can reproduce it. Thank you!
